.changeset/spotty-penguins-sort.md

```diff
@@ -0,0 +1,8 @@
+---
+"@spectrum-css/rating": major
```

Comment:
@pfulton I'm a bit anxious about introducing such a large bump to an S1 component with the current progress on foundations work. What do you think? Should we try to scale back the scope of this work, hold it until after foundations, or move forward as-is?

Reply:
I agree on the anxiousness around S1/S2 timing. On the plus side, though, this is a component that is _not_ currently available in SWC, so the downstream impact of a big change might not be as large as it could have been if it was already implemented there. That said: it's still a _major_, and we want to respect what that means for all potential consumers.

I think we should pause on this one for now until after Foundations goes in. After that, we can work through rebasing this one and getting it merged in. The scope of changes are only within this particular component, so hopefully any potential conflicts won't be too terrible to resolve when the time comes.